### ISSUE TITLE
Inconsistency problem with settings.db and database module

### ISSUE THREAD

**Issue by toby63** [FreeTubeApp/FreeTube#1504]
**Behavior of the program**
With every new start (or new window) the hide options (in this case: trending and popular videos) are ignored and also the subscribed channels are not shown (see Screenshot below).

I have to click on one of these options each time, then the correct settings are loaded and the correct UI is shown again.

**Expected behavior**
Show correct UI on start, according to settings.

**To Reproduce**
Start FreeTube or a new window of Freetube.

**Screenshots**
Excerpt of window:
![freetube-menu](https://user-images.githubusercontent.com/61703250/124974419-0f504100-e02d-11eb-9d21-3afdbf2aa171.png)

**Environment Information:**
 - OS: Arch Linux
 - OS Version: recent
 - FreeTube version: Development Build, commit gb8a5f4ac
 - Installation Method (If applicable): AUR package [freetube-git](https://aur.archlinux.org/packages/freetube-git/)
 - Primary API Used: -

**Comment by GilgusMaximus** [comment]
Can someone else report this issue? I have problems recreating this

**Comment by toby63** [comment]
Just to add some information:
- It works correctly with Release 0.13.2
- it does not work with commits gb8a5f4ac and https://github.com/FreeTubeApp/FreeTube/commit/0dbec11702b9d435f22fc443cb85cb14593a01c0 (which is the last built I tried)

**Comment by hsvallfors** [comment]
I also have this issue, likewise on nightly not release. It's still present in build [1008](https://github.com/FreeTubeApp/FreeTube/actions/runs/1069155318).

**Comment by GilgusMaximus** [comment]
Could you please check the settings.db file in the Freetube data folder and post it's content here? Maybe something is messed up in there that leads to this problem

**Comment by hsvallfors** [comment]
`{"value":false,"_id":"autoplay"}
{"value":false,"_id":"autoplayPlaylists"}
{"_id":"autoplayVideos","value":false}
{"_id":"backendPreference","value":"local"}
{"_id":"bounds","value":{"x":0,"y":24,"width":1680,"height":985,"maximized":true}}
{"value":false,"_id":"debugMode"}
{"_id":"defaultProfile","value":"All Channels"}
{"_id":"defaultQuality","value":"720"}
{"_id":"defaultTheatreMode","value":false}
{"_id":"disableSmoothScrolling","value":true}
{"value":false,"_id":"distractionFreeMode"}
{"_id":"forceLocalBackendForLegacy","value":false}
{"_id":"hidePopularVideos","value":true}
{"_id":"hideRecommendedVideos","value":false}
{"_id":"hideTrendingVideos","value":true}
{"_id":"hideWatchedSubs","value":true}
{"value":true,"_id":"history"}
{"value":"https://invidious.tube","_id":"invidious"}
{"_id":"invidiousInstance","value":"https://invidious.kavin.rocks"}
{"value":true,"_id":"localScrape"}
{"_id":"playNextVideo","value":false}
{"value":"dash","_id":"player"}
{"_id":"profileList","value":[{"name":"All Channels","color":"#FFD600"}]}
{"value":"SOCKS5://127.0.0.1:9050","_id":"proxy"}
{"value":false,"_id":"proxyVideos"}
{"value":"1080","_id":"quality"}
{"value":"1","_id":"rate"}
{"value":"US","_id":"region"}
{"value":"subscriptions","_id":"startScreen"}
{"value":false,"_id":"subtitles"}
{"value":"dark","_id":"theme"}
{"value":true,"_id":"updates"}
{"_id":"useRssFeeds","value":true}
{"_id":"useSponsorBlock","value":true}
{"value":false,"_id":"useTor"}
{"value":"grid","_id":"videoView"}
{"value":"1","_id":"volume"}`

**Comment by hsvallfors** [comment]
```
{"value":false,"_id":"autoplay"}
{"value":false,"_id":"autoplayPlaylists"}
{"_id":"autoplayVideos","value":false}
{"_id":"backendPreference","value":"local"}
{"_id":"bounds","value":{"x":0,"y":24,"width":1680,"height":985,"maximized":true}}
{"value":false,"_id":"debugMode"}
{"_id":"defaultProfile","value":"All Channels"}
{"_id":"defaultQuality","value":"720"}
{"_id":"defaultTheatreMode","value":false}
{"_id":"disableSmoothScrolling","value":true}
{"value":false,"_id":"distractionFreeMode"}
{"_id":"forceLocalBackendForLegacy","value":false}
{"_id":"hidePopularVideos","value":true}
{"_id":"hideRecommendedVideos","value":false}
{"_id":"hideTrendingVideos","value":true}
{"_id":"hideWatchedSubs","value":true}
{"value":true,"_id":"history"}
{"value":"https://invidious.tube","_id":"invidious"}
{"_id":"invidiousInstance","value":"https://invidious.kavin.rocks"}
{"value":true,"_id":"localScrape"}
{"_id":"playNextVideo","value":false}
{"value":"dash","_id":"player"}
{"_id":"profileList","value":[{"name":"All Channels","color":"#FFD600"}]}
{"value":"SOCKS5://127.0.0.1:9050","_id":"proxy"}
{"value":false,"_id":"proxyVideos"}
{"value":"1080","_id":"quality"}
{"value":"1","_id":"rate"}
{"value":"US","_id":"region"}
{"value":"subscriptions","_id":"startScreen"}
{"value":false,"_id":"subtitles"}
{"value":"dark","_id":"theme"}
{"value":true,"_id":"updates"}
{"_id":"useRssFeeds","value":true}
{"_id":"useSponsorBlock","value":true}
{"value":false,"_id":"useTor"}
{"value":"grid","_id":"videoView"}
{"value":"1","_id":"volume"}
```

**Comment by hsvallfors** [comment]
I tried renaming the settings file - This fixed the issue, whatever it was. Thanks @GilgusMaximus!

**Comment by GilgusMaximus** [comment]
Yeah, there seems to be an inconsistency problem with the database. That is the reason I asked you to change it, because when I opened mine after closing the app, some of the hide elements were duplicated.

I am gonna keep this open and rename the issue to reflect the problem better :)

Happy that this worked for you atm

Best Regards,
Gilgus

**Comment by toby63** [comment]
Experienced this again with new version 0.14.0 Beta.

**Comment by ChunkyProgrammer** [comment]
should be fixed in #1833
